File: 1574-maximum-product-of-two-elements-in-an-array/maximum-product-of-two-elements-in-an-array.java
class Solution {
    public int maxProduct(int[] nums) {
        int f_biggest = (int) Float.NEGATIVE_INFINITY;
        int s_biggest = (int) Float.NEGATIVE_INFINITY;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] > f_biggest) {
                int tmp = f_biggest;
                f_biggest = nums[i];
                s_biggest = tmp;
            } else if (nums[i] > s_biggest) {
                s_biggest = nums[i];
            }
        }
        return (f_biggest - 1) * (s_biggest - 1);
    }
}

File: 1574-maximum-product-of-two-elements-in-an-array/maximum-product-of-two-elements-in-an-array.py
class Solution:
    def maxProduct(self, nums: List[int]) -> int:
        f_biggest, s_biggest = -math.inf, -math.inf
        for i, n in enumerate(nums):
            if n > f_biggest:
                f_biggest, s_biggest = n, f_biggest
            elif n > s_biggest:
                s_biggest = n
        return (f_biggest - 1) * (s_biggest - 1)